packages/sync/src/Replicastore.php

```diff
@@ -642,48 +647,60 @@ public function checksum_all() {
 		);
 	}
 
+	function get_columns_for_object_type( $object_type ) {
+		switch( $object_type ) {
+			case "post_meta":
+				return Defaults::$default_post_meta_checksum_columns;
+			case "comments":
+				return Defaults::$default_comment_checksum_columns;
+			case "comment_meta":
+				return Defaults::$default_post_meta_checksum_columns;
+			case "terms":
+				return Defaults::$default_term_checksum_columns;
+			default:
```

Comment:
🤔 Shouldn't we ignore any other object types because we are familiar only with these?

Reply:
Possibly, however, prior to this change we used `Defaults::$default_post_checksum_columns;` as a default.